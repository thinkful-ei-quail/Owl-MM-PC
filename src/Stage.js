import React from 'react';
import ReactDOM from 'react-dom';
import particiants from './participantsDummy';
import './Stage.css';

export default function Stage(props){
    if(props.onStage){
    return (
        <div>
        <img src={props.avatar}/>
        <p>{props.name}</p>
        </div>
    )
    };
};
