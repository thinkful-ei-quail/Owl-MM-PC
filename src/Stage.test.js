import React from 'react';
import ReactDOM from 'react-dom';
import Stage from './Stage';
import renderer from 'react-test-renderer';

it('generate the stage elements',()=>{
    const div = document.createElement('div');
    ReactDOM.render(<Stage avatar="testimage" name="Matt" />,div)
    ReactDOM.unmountComponentAtNode(div);
});
it('snapshot of working UI', () => {
    const tree = renderer
        .create(<Stage avatar="testimage" name="Matt" />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});