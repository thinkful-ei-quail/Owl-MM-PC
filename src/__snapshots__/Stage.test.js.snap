// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot of working UI 1`] = `
<div>
  <img
    src="testimage"
  />
  <p>
    Matt
  </p>
</div>
`;
